src/reducers/LoaderReducer.js

```diff
@@ -0,0 +1,21 @@
+import { ENABLE_LOADER, DISABLE_LOADER } from 'actions';
+
+const LoaderReducer = (state = { loader: false }, { type }) => {
```

Comment:
same here - this can be a simple boolean, not an object with boolean inside, doesn't it ?